runtime/src/main/scala/com/soundcloud/twinagle/JsonClientFilter.scala

```diff
@@ -12,6 +12,9 @@ private[twinagle] class JsonClientFilter[
 ](
     path: String
 ) extends Filter[Req, Rep, Request, Response] {
+
+  val jsonParser = new Parser().ignoringUnknownFields
```

Comment:
`private val`?

Reply:
good point :+1:. JsonClientFilter is private itself, but yes, there's no need to expose this.